### DISCUSSION TITLE
CNN US is not working, says that the media file couldn't be opened.

### DISCUSSION THREAD

**Discussion by randomguyfromAustria** [iptv-org/community#52] [Q&A]
Hello,
I am having a problem. When i try clicking "CNN (1080p)" it says that it couldnt open the media file.
![vlc_VOAZrx1GX7](https://user-images.githubusercontent.com/95542560/189215452-550ee02a-75f4-4ca3-ba3c-65e6b71d0442.png)
Can anyone comfirm this issue or am i the only one having this issue?
Please fix this.

**Comment by ghost** [comment]
Why don't you use this link instead: https://tve-live-lln.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8

**Reply by randomguyfromAustria** [reply]
Thx it works. Do you have a link for the HD channel for CNN US or is this link the HD one?

**Reply by ghost** [reply]
This is the HD one

**Reply by f2e6** [reply]
> Why don't you use this link instead: https://tve-live-lln.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8

Is there an index of thes warnermediacdn links?

**Reply by BellezaEmporium** [reply]
It seems it doesn't. I can make an adaptive one out of those though, that I could add over iptv_exception
